third_party/2and3/click/types.pyi

```diff
@@ -55,7 +55,7 @@ class IntParamType(ParamType):
 class IntRange(IntParamType):
     def __init__(self, min: Optional[int] = ..., max: Optional[int] = ..., clamp: bool = ...) -> None: ...
 
-_PathType = TypeVar("_PathType", str, bytes)
+_PathType = Any
```

Comment:
```suggestion
_PathType = AnyStr
```

Reply:
`_PathType` is never used as a type variable, it's just a glorified `Union[str, bytes]` return type. (Which means I can leave my obligatory link to python/typing#566 here.)